geowebcache/wmts/src/main/java/org/geowebcache/service/wmts/WMTSService.java

```diff
@@ -65,6 +73,9 @@
     
     private GeoWebCacheDispatcher controller = null;
 
+    @Autowired(required = false)
```

Comment:
Normally GeoWebCache loads extension points using GeoWebCacheExtensions instead of autowiring. In GWC the mechanism is not all that different from autowiring for the moment, but eventually it could grow similar to GeoServer where there is the ability to declare extensions filters and control load order via priority.